src/context/ContextTray.jsx

```diff
@@ -1,37 +1,37 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import TemplateViewModeContent from './TemplateViewModeContent';
+import SnippetViewModeContent from './SnippetViewModeContent';
 import ShortcutViewModeContent from './ShortcutViewModeContent';
 import PlaceholderViewModeContent from './PlaceholderViewModeContent';
 import './ContextTray.css';
 
 export default class ContextTray extends Component {
     //these are the possible view mode constants
-    TEMPLATE_VIEW = 0;
+    SNIPPET_VIEW = 0;
     SHORTCUT_VIEW = 1;
     PLACEHOLDER_VIEW = 2;
 
     constructor(props) {
         super(props);
-        const viewMode = this.props.showTemplateView ? this.TEMPLATE_VIEW : this.SHORTCUT_VIEW;
+        const viewMode = this.SHORTCUT_VIEW;
         this.state = {
             // viewMode keeps track of which context is active
-            // 0 when Templates are selected. 1 when Shortcuts is selected. 2 when Placholder is selected
+            // 0 when snippets are selected. 1 when Shortcuts is selected. 2 when Placholder is selected
             // In editor, viewMode is incremented by 1 for each context added (i.e @condition, #disease status)
             viewMode,
         };
     }
 
-    componentWillReceiveProps = (nextProps) => {
-        // If show template view is true, set the view to template view, otherwise check if current view is template view. If the view is currently a template go to shortcut view, else leave the view as is
-        // (ex. if in placeholder view, stay in placeholder view)
-        const viewMode = nextProps.showTemplateView ? this.TEMPLATE_VIEW : (this.state.viewMode === this.TEMPLATE_VIEW ? this.SHORTCUT_VIEW : this.state.viewMode);
-        this.setState ({ viewMode });
-    }
+    // componentWillReceiveProps = (nextProps) => {

```

Comment:
Do we still need this old method? Is there information in here we want to keep around?